### DISCUSSION TITLE
How to add dependency changes to release notes?

### DISCUSSION THREAD

**Discussion by gerhard** [miniscruff/changie#510] [Q&A] [answered]
Is there a way to add automatically dependency changes to release notes?

This is what I'm thinking ([FTR](https://github.com/siderolabs/talos/releases/tag/v1.4.6)):
<img width="995" alt="image" src="https://github.com/miniscruff/changie/assets/3342/e2a77239-bde3-4d0b-8379-07918e759ee8">

I know that this is out of scope for `changie`, but I'm wondering how others are doing / thinking about this.

---
cc @sheldonhull @smira @frezbo

**Comment by miniscruff** [comment, marked as answer]
I know @emmyoop has done some work around dependencies in https://github.com/dbt-labs/dbt-core and related projects. You may get some insights from there [.changie.yaml](https://github.com/dbt-labs/dbt-core/blob/main/.changie.yaml) just be warned it is probably the most complex one I have seen and they are part of the reason changie has some of the options it does :smirk:

**Reply by gerhard** [reply]
I see what you mean! That looks like a good start for what we need. Thank you!

**Comment by smira** [comment]
The screenshot is a result of work of https://github.com/siderolabs/release-tool, which itself is a fork of https://github.com/containerd/release-tool

**Reply by gerhard** [reply]
Super helpful, thank you!

**Comment by sheldonhull** [comment]
I originally thought to add entries for dependencies, but I stopped as it felt counter to what changie was meant to do. Could be cool!

I guess if a changelog (which I think is better communicated as release notes) is focused on the human communication, then the only time a "chore(deps)" style entry would need to be there is if it impacts usability, features, etc. Otherwise, that's the point of the git log. 

Anyway, just some thoughts as I've been mulling over similar challenges internally. Trying to clearly seperate out release vs git log oriented audience, and deps feels best suited for git log rather than release notes.
